README.md

```diff
@@ -174,13 +174,29 @@ As reviewers, a PR should be only approved if following conditions are met (In o
 3. Changes are covered by tests (There could be few exceptions)
 
 ## Git commands cheatsheet
-
--   git clone \<repo url\>
--   git config
--   git status
--   git branch \<branch name\>
--   git checkout \<branch name\>
--   git add \<file or folder\>
--   git commit -m \<meaningful commit message\>
--   git push
--   git pull
+- git clone \<repo url\>
+- git config
+- git status
+- git branch \<branch name\>
+- git checkout \<branch name\>
+- git add \<file or folder\>
+- git commit -m \<meaningful commit message\>
+- git push
+- git pull
+
+## Database connection
+1. Configure the service account (in first iterations we will use a default credential)
+2. Ask for the database key and install it.
+3. Create a folder inside the folder where you put the project cloned folder and named it keys.
+4. Inside the keys folder move the json key.
+5. For initilize the app:
+
+import serviceAccount from `'../../../keys/metal-slug-maker-firebase-adminsdk-8dsbv-e517ebfe1a.json'`;
```

Comment:
I think it should be in the databaseLibrary file

Reply:
No, but i will skip the key file name